toasty/image.py

```diff
@@ -454,47 +485,41 @@ def from_pil(cls, pil_img):
         return inst
 
     @classmethod
-    def from_array(cls, mode, array):
+    def from_array(cls, array, wcs=None, default_format=None):
         """Create a new Image from an array-like data variable.
 
         Parameters
         ----------
-        mode : :class:`ImageMode`
-            The image mode.
         array : array-like object
-            The source data.
+            The source data. This should either be a 2-d floating point array a
+            3-d floating-point or uint8 array with shape (3, ny, nx), or a 3-d
+            uint8 array with shape (4, ny, nx).
+        wcs : :class:`~astropy.wcs.WCS`
+            The WCS coordinate system for the image.
+        default_format : str
+            The default format to use when writing the image if none is
+            specified explicitly. If not specified, this is automatically
```

Comment:
Doc text should mention that these are optional.

Reply:
Fixed